sdk/communication/Azure.Communication.Sms/tests/samples/Sample1_SmsClient.cs

```diff
@@ -4,15 +4,113 @@
 using System;
 using Azure.Core;
 using Azure.Identity;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
 
 namespace Azure.Communication.Sms.Tests.samples
 {
     /// <summary>
-    /// Samples that are used in the README.md file.
+    /// Samples to send sms
     /// </summary>
     public partial class Sample1_SmsClient
     {
-        public SmsClient CreateSmsClient()
+        [Test]
+        [SyncOnly]
+        public void SendSms()
+        {
+            SmsClient smsClient = CreateSmsClient();
+
+            #region Snippet:SendSms
```

Comment:
Now that we have these new samples, you can cleanup/remove the extra snippets from the `SmsClientLiveTests` and use these new snippets in the ReadMe file.